xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -895,72 +895,6 @@ For all other token types, the reader does not move. After the next call to <xre
 The current depth exceeds the recursive limit set by the maximum depth.</exception>
       </Docs>
     </Member>
-    <Member MemberName="TextEquals">
```

Comment:
Since we are updating this to remove `TextEquals` (and losing the documentation that came with it), can we add the documentation for all the `ValueTextEquals` overloads? They are there in source. But I think that can be done separately in a subsequent PR. Presumably, your tool should pick it up, right @carlossanlop 

https://github.com/dotnet/corefx/blob/fc8bd8ec1c622dd27f96d05f1bf05909f3b3cde0/src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.cs#L412-L432

Reply:
I think we should get all the complex suggestions addressed in a separate PR.